src/public/index.html

```diff
@@ -7,10 +7,34 @@
     <meta name="description" content="A blog about something awesome!">
     <title>My Blog</title>
   </head>
+  <header>
+    <h1>Something Awesome!</h1>
+  </header>
+
 
     <body>
-      <h1> Yoooooo </h1>
-      <h2> Y'all ready for this?</h2>
+      <nav>
+        <ul>
+          <a href="/About/"> About
```

Comment:
An unordered list (`<ul>`) must contain list items (`<li>`). The anchor tags (`<a>`) should be nested inside  `<li>` tags, and all `<li>` tags collectively should be wrapped inside one set of `<ul>` tags.